subxt/src/backend/unstable/mod.rs

```diff
@@ -321,7 +321,9 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
             .events()
             .filter_map(|ev| {
                 let out = match ev {
-                    FollowEvent::Initialized(init) => Some(init.finalized_block_hash.into()),
+                    FollowEvent::Initialized(init) => {
+                        init.finalized_block_hashes.last().map(|b| b.clone().into())
```

Comment:
we could refactor latest_finalized_block_ref to return all block_hashes but this should be equivalent to the old code.

Reply:
Yeah, I don't think there's any reason to report the older ones anyway; we just want to know the most recent one so that we can submit txs etc with it in mind :)